mapstruct-spi-accessor-naming/pom.xml

```diff
@@ -53,66 +52,23 @@
                 <version>${org.mapstruct.version}</version>
             </dependency>
 
-            <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>spi</artifactId>
-                <version>${project.version}</version>
-            </dependency>
-
            <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
                 <version>4.12</version>
             </dependency>
-
         </dependencies>
     </dependencyManagement>
 
     <build>
-
         <pluginManagement>
             <plugins>
-                <!-- leave annotation processing to the maven-processor-plugin to avoid duplicates -->
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.5.1</version>
-                    <configuration>
```

Comment:
This is what I like 👍 .. Now NetBeans also picks up annotations automatically (only works with the standard maven-compiler-plugin..